.github/workflows/build_fastdds.yml

```
@@ -89,19 +103,31 @@ jobs:
         with:
           cmake_build_type: ${{ matrix.cmake_build_type }}
 
-      - name: Get colcon.meta and .repos files to build fastdds
-        uses: eProsima/eProsima-CI/multiplatform/get_configurations_from_repo@main
+      - name: Get colcon.meta file to build Fast DDS
+        uses: eProsima/eProsima-CI/multiplatform/get_file_from_repo@main
         with:
-          source_repository_branch: ${{ inputs.built_configuration_branch || env.default_configuration_branch }}
-          colcon_meta_file_path: .github/workflows/configurations/metas/${{ matrix.os }}/colcon.meta
-          repos_file_path: .github/workflows/configurations/fastdds/dependencies.repos
-          colcon_meta_file_result: ${{ github.workspace }}/colcon.meta
-          repos_file_result: ${{ github.workspace }}/dependencies.repos
-
-      - name: Build Fast DDS and generate artifact
-        uses: eProsima/eProsima-CI/multiplatform/generate_dependency_artifact@main
+          source_repository_branch: ${{ env.default_configuration_branch }}
```

If the environment variable is removed:

```suggestion
          source_repository_branch: main
```

---

Env variable not removed